templates/terraform/processing-project/bigquery.tf

```diff
@@ -0,0 +1,17 @@
+resource google_bigquery_dataset dataset {
+  provider = google.target
+  dataset_id = "bigquery-dataset-dev"
+  friendly_name = "BigQuery Dev Dataset"
```

Comment:
I think I'd just leave this out, I don't see much value in it over `dataset_id` + `description`